evm/its.js

```diff
@@ -539,6 +617,7 @@ if (require.main === module) {
 
     addExtendedOptions(program, { address: true, salt: true });
 
+    program.addOption(new Option('-c, --contractName <contractName>', 'contract name').default('InterchainTokenService'));
```

Comment:
Can also add via addExtendedOptions, it will be mandatory but there won't be a default value:
`addExtendedOptions(program, { address: true, salt: true, contractName: true });`

Reply:
the goal was to have the default value, and keep the flag for consistency (it's used by `mainProcessor` to skip chains where the contract doesn't exist)

Reply:
that makes sense